proto/decentraland/social/friendships/friendships.proto

```diff
@@ -99,19 +104,29 @@ message Payload {
   optional string synapse_token = 1;
 }
 
+message ServiceResponse {
```

Comment:
Why would we want to make the response so generic when we know the actual expected type in each case?

Reply:
So you're suggesting something like:

```
message GetFriendsResponse {
  oneof response {
    ServiceErrorCode error = 1;
    Users friends = 2;
  }
}

message GetRequestEventsResponse {
  oneof response {
    ServiceErrorCode error = 1;
    RequestResponse friends = 2;
  }
}
```
And so on?

Reply:
I prefer that approach. It will have duplication on the protocol side but will be more descriptive of the actual API, more granular specs and will be safer and clearer for implementations.

Currently I don't know of the existence of generics in the protobuf language, which ultimately would be a better choice.

Reply:
I reverted the change since we're going to use [this](https://github.com/decentraland/rpc-rust/blob/main/rpc/src/rpc_protocol/index.proto#L64) and @lauti7 will be working [here](https://github.com/decentraland/rpc-rust/issues/32) to enable it.